src/components/dialogs/network-modifications/static-var-compensator/creation/susceptance-area.tsx

```diff
@@ -0,0 +1,177 @@
+/**
+ * Copyright (c) 2024, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+
+import Grid from '@mui/material/Grid';
+import {
+    AUTOMATON,
+    B0,
+    CHARACTERISTICS_CHOICE_AUTOMATON,
+    CHARACTERISTICS_CHOICES,
+    MAX_Q_AT_NOMINAL_V,
+    MAX_SUSCEPTANCE,
+    MIN_Q_AT_NOMINAL_V,
+    MIN_SUSCEPTANCE,
+    Q0,
+    SETPOINTS_LIMITS,
+    SLIDER_Q_NOMINAL,
+    SLIDER_SUSCEPTANCE,
+} from 'components/utils/field-constants';
+import { SliderInput, TextInput } from '@gridsuite/commons-ui';
+import { gridItem, ReactivePowerAdornment, SusceptanceAdornment } from '../../../dialogUtils';
+import React, { useEffect } from 'react';
+import { useFormContext, useWatch } from 'react-hook-form';
+import { getFloatNumber } from './standby-automaton-form-utils';
+import { isValidPercentage } from '../../../percentage-area/percentage-area-utils';
+import { InputAdornment, TextField } from '@mui/material';
+import { FormattedMessage } from 'react-intl';
+
+export const SusceptanceArea = () => {
+    const id = AUTOMATON;
+    const { setValue } = useFormContext();
+    const watchChoiceAutomaton = useWatch({ name: `${id}.${CHARACTERISTICS_CHOICE_AUTOMATON}` });
+    const minS = useWatch({ name: `${SETPOINTS_LIMITS}.${MIN_SUSCEPTANCE}` });
+    const maxS = useWatch({ name: `${SETPOINTS_LIMITS}.${MAX_SUSCEPTANCE}` });
+    const minQ = useWatch({ name: `${SETPOINTS_LIMITS}.${MIN_Q_AT_NOMINAL_V}` });
+    const maxQ = useWatch({ name: `${SETPOINTS_LIMITS}.${MAX_Q_AT_NOMINAL_V}` });
+
+    useEffect(() => {
+        let avgSfixeValue = (getFloatNumber(minS) + getFloatNumber(maxS)) / 2;
+        let avgQfixeValue = (getFloatNumber(minQ) + getFloatNumber(maxQ)) / 2;
+        setValue(`${id}.${B0}`, getFloatNumber(avgSfixeValue));
+        setValue(`${id}.${SLIDER_SUSCEPTANCE}`, getFloatNumber(avgSfixeValue));
+        setValue(`${id}.${Q0}`, getFloatNumber(avgQfixeValue));
+        setValue(`${id}.${SLIDER_Q_NOMINAL}`, getFloatNumber(avgQfixeValue));
+    }, [setValue, minS, minQ, maxS, maxQ, id]);
```

Comment:
![CSPR_restore](https://github.com/user-attachments/assets/319d452c-f0c6-4747-90a8-f6766099b761)

BUG, we can not restore b0 or q0 value which is saved in the previous edition.
In my opinion, auto-compute b0 or q0 value should be done only when it has not value yet.